docs/sections/reusable_text.rst

```diff
@@ -0,0 +1,12 @@
+This recipe uses the ``aarch64`` (64-bit ARM) Miniforge for Linux, and installs into ``$HOME/conda``. Adjust as necessary for you    r target system.
```

Comment:
Looks like some extra space in `you    r`, and also after `Anaconda` on the line below.

Reply:
I'll update this. Sorry about that, it doesn't appear to show up on the actual documentation webpage, but only in the rst itself.